src/graph_compute/common.h

```diff
@@ -64,7 +90,7 @@
 #endif
 
 #ifndef NUMA_WORKERS
-  #define NUMA_WORKERS 1
+  #define NUMA_WORKERS 0
```

Comment:
I'm kinda confused about this -- with D1_NUMA set, NUMA_WORKERS is already defined up on L76-80. Why try to redefine it?